MANIFEST.in

```diff
@@ -0,0 +1,7 @@
+include LICENSE
+include README.md
+include pyproject.toml
```

Comment:
There are several other readmes at the root level - do we need to add those here